### ISSUE TITLE
Build error on aarch64

### ISSUE THREAD

**Issue by imzhuhl** [alibaba/DeepRec#368]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): linux ubuntu 18.04
- Python version: 3.6
- Installed using virtualenv? pip? conda?: pip
- Bazel version (if compiling from source): 0.26.1
- GCC/Compiler version (if compiling from source): 7.5
- CUDA/cuDNN version: None
- GPU model and memory: None

**Describe the problem**
Build from source failed on arm64 server when run command:
```bash
bazel build -c opt --config=opt //tensorflow/tools/pip_package:build_pip_package
```
<img width="1185" alt="image" src="https://user-images.githubusercontent.com/17999964/184098750-f8056652-d10e-479e-ace7-4f60c96072f7.png">

I just use cpu version and build it successfully on x86 machine, so it should be related to the platform.
I find no arm image either, do you have any plan to support for arm.

**Comment by liutongxuan** [comment]
good catch, we'll followup the issue as soon as possible. Of course, we'll provide arm image as well.
